### DISCUSSION TITLE
The "AUTOMATIC1111 of text generation" checklist

### DISCUSSION THREAD

**Discussion by BetaDoggo** [oobabooga/text-generation-webui#161] [General]
The list so far, I'll update it as progress is made:
1. Broad support for a large number of models ✔️
2. Extension support ✔️
3. Questionable licensing ✔️
4. Support for leaked technology ✔️
5. Massive controversy ❌ 
6. Banned from major ML discord ❌
7. Rumors about the dev's death after 2 days of inactivity ❌

It's a good start but there's still quite a ways to go.

**Comment by cmp-nct** [comment]
wait until someone asks it the wrong question publicly. Automatic started calm as well.

**Comment by St33lMouse** [comment]
I think we can achieve the rest of these important goals by writing an extension to let it send email and telegram chats on your behalf. It's well within reach.

We could also do an extension to let it create social media accounts. That extension can make use of the receive image facility to get past the Turing tests.

Finally, we put Erebus or the 4chan model in charge and assign an outrageous personality to it. The Joker, perhaps. Use the Stable Diffusion api link so it can create pictures with the Grapefruit anime model and turn it loose!

**Comment by gsgoldma** [comment]
i think the snark should be held off until the code is less buggy than month old soup

**Reply by St33lMouse** [reply]
Oh, I'm just having a laugh. Believe me, I want this project to succeed. There's no way development can get done if everyone just sits around wringing their hands about how someone might misuse it.

People WILL misuse it, that's just a simple fact, and we'll have to deal with it when we get there. If Ooba didn't do this project, eventually someone else would do something like it and the misuse would happen anyway. 

So I look on the positive side and think of all the amazing things we'll be able to do with this tech and this project. It's very exciting, and I have high hopes for it.

**Comment by krakensurf** [comment]
I'm not an avid user of git, so forgive me if this is the wrong way to submit code.  

Here's the equivalent of webui-user.bat in automatica1111 so that you can start the interface by simply double clicking on the file in windows.  I basically copied the file from Automatic and tweaked it.   Works pretty cool on my machine.

@echo off
::git pull

::set PYTHON=l:/Python310/Python.exe
set env CUDA_VISIBLE_DEVICES=1
set GIT=
set VENV_DIR=venv
::activate_venv
set PYTHON="%VENV_DIR%\Scripts\python.exe"
echo venv %PYTHON%

%PYTHON% server.py %*

**Comment by Qualzz** [comment]
add support for a leaked model. 🌊
